definitions/variable/technology/power-plant.yaml

```diff
@@ -199,26 +192,19 @@
     unit: GWh
 - Minimum Storage|Electricity|Energy Storage System:
     description: Minimum energy that can be stored by the energy storage system unit.
-    unit:
-    - MWh
-    - GWh
+    unit: GWh
 - Spillage|Electricity|Energy Storage System:
     description: Energy spillage that could not be stored by an energy storage system
       (ESS). Several energy models include spillage variables related to renewable
       energy production; however, another way is to include a spillage variable to
       the ESS unit since it is the last sink of the power network.
-    unit:
-    - MWh
-    - GWh
-- Storage Type|Electricity|Energy Storage System:
-    description: Storage type (daily, weekly, monthly, etc.) for energy storage system
-      (ESS) that includes hydro, battery, etc.
-    unit:
-    - Daily
-    - Weekly
-    - Monthly
-- Stored Energy Inventory|Electricity|Energy Storage System:
-    description: State of the stored energy by ESS
-    unit:
-    - MWh
-    - GWh
+    unit: GWh
+- Seasonality|Electricity|Energy Storage System|{Energy Storage System Type}:
+    description: seasonality (daily, weekly, monthly, etc.) of {Energy Storage System Type}
+    unit: [daily,weekly,monthly]
```

Comment:
This is not really a unit, I guess, but the allowed values? Non-numeric values are not foreseen in the IAMC timeseries data format.